sdk/serialization/azure-json/CHANGELOG.md

```diff
@@ -1,16 +1,15 @@
 # Release History
 
-## 1.3.0-beta.1 (Unreleased)
+## 1.3.0 (Unreleased)
 
 ### Features Added
-- Enabled parsing JSONC in the `DefaultJsonReader`
 
-### Breaking Changes
-
-### Bugs Fixed
+- Added the ability to configure JSONC parsing with `JsonOptions`.
```

Comment:
nit: JsonOptions.setJsoncSupported

Reply:
I'll add this in a follow-up PR where I only change the README as CI passed and a README only change shouldn't have as much scrutiny on skipping CI